solidstart-tanstackquery-tailwind-modules/src/root.tsx

```diff
@@ -37,9 +38,11 @@ export default function Root() {
         <QueryClientProvider client={queryClient}>
           <Suspense>
             <ErrorBoundary>
-              <Routes>
-                <FileRoutes />
-              </Routes>
+              <Layout>
```

Comment:
The layout will be a problem because it means signin page will have the header too. There has to be a way solid start handles layouts maybe

Reply:
We can also choose to tackle this when the auth is merged, depending on you and we can do a little research and if it's taken too long we can merge and do a follow-up ticket. What do you think?

Reply:
yes, I think this is the best thing to do. Merge the auth branch and then see how to manage the common layout. I took a look at solid start docs, but it seems there is no best practice to manage layout. They have a `<Outlet>` component that could be used but using the routing with `<A>` link makes it impossible I think. But we can definitely take another look at this after the auth branch is merged.